sycl/unittests/event/GetWaitList.cpp

```diff
@@ -0,0 +1,54 @@
+//==--------------------- GetWaitList.cpp --- event unit test --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+#include <gtest/gtest.h>
+
+TEST(GetWaitList, GetWaitListTest) {
+  sycl::queue q = sycl::queue(sycl::default_selector());
+
+  sycl::event eA = q.submit([&](sycl::handler &cgh) {
+    cgh.host_task([]() {
+      double p = 0;
+      p++;
+    });
```

Comment:
```suggestion
    cgh.host_task([]() {});
```
Can we simplify host task to just a lambda with an empty body?

Reply:
I simplified it